### PR TITLE
Add Fedora 36 to CI

### LINKED ISSUE #1825: Fedora 36, please.
Could someone generate MAVSDK for Fedora 36, please.

Thanks

### PR DESCRIPTION
**Pull request by JonasVautherin**
Let's see...

Resolves #1825.

### CODE REVIEW

**Comment by JonasVautherin** [comment]
Oh, there is actually a need to create the docker image :thinking:

File: docker/Dockerfile-Fedora-36 [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#
+# Development environment for the MAVSDK based on Fedora 36.
+#
+# Author: Julian Oes <julian@oes.ch>
```
Comment (julianoes) [inline docker/Dockerfile-Fedora-36:4]: Debatable...

File: docker/Dockerfile-Fedora-36 [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+#
+
+FROM fedora:36
+
+MAINTAINER Julian Oes <julian@oes.ch>
```
Comment (julianoes) [inline docker/Dockerfile-Fedora-36:9]: :man_shrugging:

**Review by julianoes — APPROVED** [review]
Nice! Did you generate just this one or build and push all of them?

**Pull request merged**